.pylintrc

```diff
@@ -535,7 +535,7 @@ max-bool-expr=5
 max-branches=22
 
 # Maximum number of locals for function / method body.
-max-locals=26
+max-locals=27
```

Comment:
We should stop doing this, we've gone too far from 15

Reply:
the problem is the import statement, every import inside a function is a "new local variable"

Reply:
changed the function and returned the original value on `.pylintrc`